applications/ALEapplication/python_scripts/mesh_solver_base.py

```diff
@@ -192,9 +192,12 @@ def _set_and_fill_buffer(self):
         step =-buffer_size
         time = time - delta_time * buffer_size
         self.mesh_model_part.ProcessInfo.SetValue(KratosMultiphysics.TIME, time)
-        for i in range(0, buffer_size):
+        #for i in range(0, buffer_size):
```

Comment:
actually i think i preferred your first version for this one ;-) (accepting)

Reply:
does codacy propose ```while``` loops instead of ```for i in range(...)``` ?